Doc/library/subprocess.rst

```diff
@@ -797,6 +800,19 @@ on Windows.
       :data:`SW_HIDE` is provided for this attribute. It is used when
       :class:`Popen` is called with ``shell=True``.
 
+   .. attribute:: lpAttributeList
+
+      A dictionary of additional attributes for process creation as given in
+      ``STARTUPINFOEX``, see
+      `UpdateProcThreadAttribute <https://msdn.microsoft.com/en-us/library/windows/desktop/ms686880(v=vs.85).aspx>`__.
+
+      Supported attributes:
+
+      **handle_list**
+         When *close_fds* is :const:`True`, supplies a list of
+         handles that will be inherited.
```

Comment:
I suggest the doc:

"Sequence of handles that will be inherited. close_fds must be true if non-empty."

(And make sure that any sequence type is accepted.)